contracts/cw20-merkle-airdrop/src/contract.rs

```diff
@@ -0,0 +1,487 @@
+#[cfg(not(feature = "library"))]
+use cosmwasm_std::entry_point;
+use cosmwasm_std::{
+    attr, to_binary, Addr, Binary, Deps, DepsMut, Env, MessageInfo, Response, StdResult, SubMsg,
+    Uint128, WasmMsg,
+};
+
+use crate::state::{Config, CLAIM, CONFIG, MERKLE_ROOT, STAGE};
+
+use crate::error::ContractError;
+use crate::msg::{
+    ConfigResponse, ExecuteMsg, InstantiateMsg, IsClaimedResponse, LatestStageResponse,
+    MerkleRootResponse, MigrateMsg, QueryMsg,
+};
+use cw20::Cw20ExecuteMsg;
+use cw_storage_plus::U8Key;
+use sha3::Digest;
+use std::convert::TryInto;
+
+// Version info, for migration info
+const _CONTRACT_NAME: &str = "crates.io:cw20-merkle-airdrop";
```

Comment:
unused. want to add the cw2 line to store them? I forget what it is, but a one-liner from most other cosmwasm-plus contracts